ecosystem/sep-0010.md

```diff
@@ -19,56 +19,56 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
 It involves three components:
-- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`), referred to as the home domain.
-- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain where the endpoint is hosted is referred to as the web auth domain. 
-- A client who will authenticate using a Stellar account.
+- A **Home Domain**: a domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`). The ``WEB_AUTH_ENDPOINT`` must be hosted by a **Web Auth Endpoint** as described below.
+- A **Web Auth Endpoint**: a domain hosting an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on a **Home Domain**, a sub-domain of a **Home Domain**, or any other domain.
```

Comment:
❗ The endpoint isn't a domain, but it leads by saying that it is a domain.

```suggestion
- A **Web Auth Endpoint**: an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the **Home Domain**, a sub-domain of the **Home Domain**, or any other domain.
```

❗ After re-reading this I think it is still necessary to either keep the sentence `The domain where the endpoint is hosted is referred to as the web auth domain.` which is in the original, or to add that statement to a new list item. A list item would be more explicit:

```suggestion
- A **Web Auth Endpoint**: an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on a **Home Domain**, a sub-domain of a **Home Domain**, or any other domain.
- A **Web Auth Domain**: the domain where the **Web Auth Endpoint** is hosted. 
```

Reply:
I agree, its ultimately better to be explicit even if `Web Auth Domain` is seldom used. Editing your 2nd suggestion so that it doesn't conflict with the first -- I'm committing both.